src/authentication/login.handler.ts

```diff
@@ -89,7 +89,7 @@ export const withLogin = (
       email: string;
       password: string;
     };
-    const adminUser = await auth.authenticate(email, password);
+    const adminUser = await auth.authenticate(email, password, req);
```

Comment:
Why do you need it?

Reply:
Because I want to get the IP address of the authenticated user

Reply:
Why would you need the IP address in your `authenticate` method? This looks like a very rare case where you'd probably end up better creating your own authentication mechanism.

Reply:
hey how do we create custom authentication?

Reply:
> Why would you need the IP address in your `authenticate` method? This looks like a very rare case where you'd probably end up better creating your own authentication mechanism

I want to save the IP address of the authenticated user and maybe in the future I will want to block authentications from unknown IPs